examples/reference/publication_quality/bokeh/bar.py

```diff
@@ -0,0 +1,209 @@
+# Demonstrates:
+# - Data extraction, transformation, and visualization separation
+# - Custom Bokeh themes for consistent styling
+# - Interactive tooltips with formatted data
+# - Text annotations on bars
+# - Professional fonts, grids, and axis formatting
+# - Panel integration for web serving
+# ============================================================================
+
+import panel as pn
+import hvsampledata
+import holoviews as hv
+
+from bokeh.models.formatters import NumeralTickFormatter
+from bokeh.themes import Theme
+from holoviews.plotting.bokeh import ElementPlot
+
+# ============================================================================
+# BOKEH THEME SETUP - Define global styling
+# ============================================================================
+
+ACCENT_COLOR = '#007ACC'  # Professional blue
+FONT_FAMILY = 'Roboto'
+
+def create_bokeh_theme(font_family=FONT_FAMILY, accent_color=ACCENT_COLOR):
+    """Create custom theme with specified font. Default: Roboto"""
+    return Theme(json={
+        'attrs': {
+            'Title': {
+                'text_font': font_family,
+                'text_font_size': '16pt',
+                'text_font_style': 'bold'
+            },
+            'Axis': {
+                'axis_label_text_font': font_family,
+                'axis_label_text_font_size': '12pt',
+                'axis_label_text_font_style': 'bold',
+                'major_label_text_font': font_family,
+                'major_label_text_font_size': '10pt',
+                'major_tick_line_color': "black",  # Remove tick marks
+                'minor_tick_line_color': None
+            },
+            'Plot': {
+                'background_fill_color': '#fafafa',
+                'border_fill_color': '#fafafa'
+            },
+            'Legend': {
+                'label_text_font': font_family,
+                'label_text_font_size': '10pt'
+            },
+            'Toolbar': {
+                "autohide": True,
+                "logo": None,
+                "stylesheets": [
+                    f"""
+                    .bk-OnOffButton.bk-active{{
+                        border-color: {accent_color} !important;
+                    }}
+                    """
+                ]
+            },
+            # Does not work via Theme, so added here for reference purposes until I figure out how to do it
+            'Tooltip': {
+                "stylesheets": [f"""
+                    .bk-tooltip-row-label {{
+                        color: {ACCENT_COLOR} !important;  
+            }}"""]
+                
+            }
+        }
+    })
+
+# Apply theme globally - affects all plots
+hv.renderer('bokeh').theme = create_bokeh_theme()
+
+# ============================================================================
+# HOLOVIEWS OPTS SETUP - Define global configuration
+# ============================================================================
+
+GLOBAL_BACKEND_OPTS={
```

Comment:
I tried to set these GLOBAL_OPTS in the bokeh Theme but could not get it working. Especially the `min_border` would be nice to set in the theme instead.